airbyte-integrations/connectors/destination-clickhouse-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/clickhouse/ClickhouseDestinationStrictEncryptAcceptanceTest.java

```diff
@@ -184,11 +190,12 @@ protected void tearDown(final TestDestinationEnv testEnv) {
     db.close();
   }
 
+  @Override
   @ParameterizedTest
   @ArgumentsSource(DataTypeTestArgumentProvider.class)
   public void testDataTypeTestWithNormalization(final String messagesFilename,
-                                                final String catalogFilename,
-                                                final DataTypeTestArgumentProvider.TestCompatibility testCompatibility)
+      final String catalogFilename,
```

Comment:
Minor nit: most methods _should_ be formatted to have this all within the same column, can we revert this so that it's not on the same column as the `throws Exception`?

Reply:
done